frontend/app/controllers/Joiner.scala

```diff
@@ -170,11 +183,24 @@ trait Joiner extends Controller with ActivityTracking {
       Future.sequence(optFuture.toSeq).map(_.headOption)
     }
 
+    val futureContentOpt = request.cookies.get("GU_MEM_REFERER").map(_.value).map { referer =>
+      val refererPath = referer.replace("http://www.theguardian.com", "")
```

Comment:
Use the `Config` class to define http://www.theguardian.com. That way we can configure it for dev environments etc.